apps/condo/domains/meter/utils/testSchema/index.js

```diff
@@ -49,7 +50,7 @@ const { EXCEL } = require('@condo/domains/common/constants/export')
 const { LOCALE_EN } = require('@condo/domains/user/constants/common')
 
 
-async function createTestMeterResource (client, extraAttrs = {}) {
```

Comment:
Please tune your IDE settings.
In most cases, we use space when defining functions and not use space when calling functions.

Reply:
you're right, i confused the styling for some reason